R/completion.R

```diff
@@ -10,6 +8,14 @@ getSymbolsFromAttachedPackages <- function(text) {
   unlist(symbols, use.names = FALSE)
 }
 
+
+getPackages <- function() {
+  pkgs <- search()
```

Comment:
There's a difference for getting packages via `search()` and the usage of `pkg::` or `pkg:::`. `search()` returns the *attached* packages while `pkg::` and `pkg:::` could be any installed packages (`.packages(all.available = TRUE)` is the fastest way to get the names).